controllers/front/notify.php

```diff
@@ -181,23 +195,19 @@ public function postProcess()
                 die('canceled');
             }
 
-            PrestaShopLogger::addLog(
-                sprintf(
-                    '%s has caught an error: %s',
-                    __CLASS__,
-                    $e->getMessage()
-                ),
-                1,
-                null,
-                null,
-                null,
-                true
-            );
+            /** @var LoggerInterface $logger */
+            $logger = $this->module->getService(LoggerInterface::class);
+            $logger->error(sprintf('%s - caught an error: %s', self::FILENAME, $e->getMessage()), [
+                'exception' => $e,
```

Comment:
Exception is logged like this
            $logger->error('Failed to do something', [
                'context' => [],
                'exceptions' => ExceptionUtility::getExceptions($exception),
            ]);